docs/cmdline-opts/upload-flags.md

```diff
@@ -0,0 +1,23 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+SPDX-License-Identifier: curl
+Long: upload-flags
+Arg: <flags>
+Help: Specify additional upload behavior
+Category: curl output
+Added: 8.13.0
+Multi: single
+See-also:
+  - upload-file
+Example:
+  - --upload-flags Flagged,!Seen --upload-file local/dir/file $URL
+---
+
+# `--upload-flags`
+
+Specify additional behavior to apply to uploaded files. Flags are
+specified as either a single flag value or a comma-separated list
+of flag values. Flag values may be negated by prepending them with
+a '!' character. Currently the following case-sensitive flag values
```

Comment:
For the `--proto` option, we use `-` (minus) for removing a protocol. Since this is a very similar syntax, don't you think it would make sense to use minus here as well for consistency?